contents/uix/accessibility.md

```diff
@@ -0,0 +1,141 @@
+  ![Sign up page with more visual indicators](accessibility/fb-more-indicators.png "Facebook sign up page with more visual indicators")
+
+  * Maintain good contrast
+  
+  Look at this page taken from Tech Crunch.
+  ![Tech Crunch page](accessibility/tc-normal.png "Tech Crunch page")
+
+  To users with good vision, the design may look minimalist and clean. However, to someone suffering from cataract, the page looks like this.
+
+  ![Blurry Tech Crunch page](accessibility/tc-blurred.png "Tech Crunch page to someone with cataract")
+  It gets difficult to read the news snippets because of the poor color contrast (grey on white). On the other hand, the black colored texts are still legible. Thus, we should maintain good contrast ratio in our websites for ease of reading.
+
+  * Use `alt` attribute for images
+  
+  The `alt` attribute provides alternative information for an image that can be read out by a screen reader to describe an image.
+
+  ![Use of alt tag](accessibility/alt-tag.png "Alt tag")
+
+  
+### Auditory
+Hearing-impaired users do use assistive technologies such as a hearing aid or a cochlear implant, but these are not specific for accessing websites.
+
+For users with hearing impairment, we should provide text alternatives to audio content, such as:
+  * Text transcripts
+  * Captions
+
+  
+### Cognitive
+Cognitive disabilities range from mental illnesses to learning difficulties, difficulties in comprehension and concentration, etc. Some examples include ADHD (attention deficit hyperactivity disorder), and autism.
+
+Such disabilities might affect website usage. It could be due to difficulty in understanding how to complete a task, remembering how to do something that was previously accomplished, increased frustration at confusing workflows or inconsistent layouts/navigation/other page features.
+
+Unlike other web accessibility issues, there is no quick fixes to issues arising from cognitive disabilities. The rule of thumb you can follow is to always design your websites to be as logical, consistent, and usable as possible. Here is a checklist that you can follow:
+
+* Pages are consistent — navigation, header, footer, and main content are always in the same places.
+* Tools are well-designed and easy to use.
+* Multi-stage processes are broken down into logical steps, with regular reminders of how far through the process you are, and how long you've got left to complete the process, if appropriate.
+* Workflows are logical, simple, and require as few interactions as possible to complete. For example, registering and signing in to a website is often unneccessarily complex.
+* Pages are not overly long or dense in terms of the amount of information presented at once.
+* The language used in your pages is as plain and easy to follow as possible, and not full of unneccessary jargon and slang.
+* Important points and content are highlighted in some way.
+* User errors are clearly highlighted, with help messages to suggest solutions.
```

Comment:
I feel that these are all the techniques we should use to design a website even without thinking about cognitive disability people? Is there any different from the normal way that we need to take note specifically? Like too many points and I feel there is no distinct part.